packages/client/hmi-client/src/model-representation/mira/mira.ts

```diff
@@ -33,8 +33,9 @@ export const emptyMiraModel = () => {
 /**
  * Collection of MMT related functions
  * */
-export const getContextKeys = (miraModel: MiraModel) => {
```

Comment:
You could keep this function with the following, so you do not have to modifed the service. Could still be useful
```javascript
export const getContextKeys = (miraModel: MiraModel) => getContext(miralModel).keys;
```